### ISSUE TITLE
Review Facility Information page

### ISSUE THREAD

**Issue by suhafa** [bcgov/cas-reporting#195]
#### Description:
"As an Industry Reporter, I want a step to review my Facility information so that it is accurate for the current report"

After reviewing Operation-level information, Industry Reporters need to review some facility-level information to ensure it is accurate for the purposes of filing a report. Much of this information (if not all?) comes from Registration. 
For LFO's in particular, the list of facilities (#197 ) needs to be up to date in order to properly review information at this step. 
It may look different for SFOs. Discussions are on-going. 

Facility Information includes: 

- Facility Name
- Facility Type
- Facility BCGHGID (non-editable)
[removed list of products]
- List of Activities (with relevant ones selected as chosen in registration)

[Wireframe](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7826-16179&t=wSMq8kpVYHYUnlLr-4)

#### Acceptance Criteria:

Given I am a reporter
When I start a report, and have already reviewed Operation Information and my list of facilities (if applicable)
Then the first step of the report is to confirm my facility information

Given I am a reporter
When I confirm my facility information
Then the facility information I just confirmed is included in the report

#### Development Checklist:

- [x] ~Report versioning scheme is figured out~ done in #196 
- [x] Facility Name & type can be edited
- [ ] Products can be checked/unchecked [NO LONGER APPLICABLE]
- [x] Checked products are saved to the `erc.report_facility_products` table [NO LONGER APPLICABLE]
- [x] Unchecked products are deleted from the `erc.report_facility_products` table [NO LONGER APPLICABLE]
- [x] Checked activities are saved to the `erc.report_facility_activities` table
- [x] Unchecked activities are deleted from the `erc.report_facility_activities` table
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by mikevespi** [comment]
It would be helpful to know a little bit more about the context / workding of what we're asking for in registration part 2. For example is the wording asking for the information that is current or relevant for the reporting period, are we asking for complete activity lists or something more loose like "main activity". The distinctions are most impactful when we look at sales and transfers, or when there's a correction to be made to something we show them.

**Comment by pbastia** [comment]
Is this for multiple facilities in an LFO? Does this need to reflect that we're confirming the entire composition of an LFO?

**Comment by pbastia** [comment]
This is not blocked by #196, I'll remove the blocker @rdromey @patriciarussellCAS

**Comment by patriciarussellCAS** [comment]
Still to be confirmed: Will this page look different for SFOs vs LFOs? 
We still want to keep this page as the first step of reporting, but if there are fields that would be redundant (i.e. they are the same in "review operation information" as they are here, we could hide them)
We should have more clarity on this after user testing.

**Comment by suhafa** [comment]
Design review (based off the sprint demo):


![Image](https://github.com/user-attachments/assets/381b60b6-5353-4954-8138-f7026eba1926)

1) The BCGHGID field has low contrast and is not legible and doesn't indicate that it is uneditable. For uneditable fields, the design needs to look like this:


![Image](https://github.com/user-attachments/assets/3baa850b-9495-4405-bea2-07c33921b00f)

2) Facility type needs to be a dropdown
3) Would products be included in a later ticket?

Thanks for your work on this @ayeshmcg !

**Comment by ayeshmcg** [comment]
Added ticket https://github.com/bcgov/cas-reporting/issues/320 
for facility type

**Comment by patriciarussellCAS** [comment]
Hi @ayeshmcg - could you share the URL for this page so I can review it? 
Also I made some notes in the follow-up ticket, so once I've reviewed this in Dev I can close this ticket. 
Thanks!

**Comment by ayeshmcg** [comment]
Hi @patriciarussellCAS ,
Here's the URL for testing: https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/reporting/reports/5/facilities/1/review

**Comment by patriciarussellCAS** [comment]
I have created a design backlog ticket for this. #326 
We also have a follow-up ticket for the facility type; should this be a drop-down or not? #320 
I will close this ticket.
